src/Cli/dotnet/commands/dotnet-workload/search/WorkloadSearchCommand.cs

```diff
@@ -34,10 +41,44 @@ public WorkloadSearchCommand(
 
             _sdkVersion = creationResult.SdkVersion;
             _workloadResolver = creationResult.WorkloadResolver;
+
+            _numberOfWorkloadSetsToTake = result.GetValue(SearchWorkloadSetsParser.TakeOption);
+            _workloadSetOutputFormat = result.GetValue(SearchWorkloadSetsParser.FormatOption);
```

Comment:
It looks like we don't do any validation that this is a valid value.  So if you have `jsno` or something as a typo it will just output it as CSV.

I don't think CSV with no spaces is great as a default.  If the default is supposed to be human-readable, I would suggest outputting it in `text` format where each version is on a separate line.  Eventually this might change to be a table.

We should think about what information we might want to include in the future and try to make sure the output formats can support that without breaking changes.  So for the json output we may want each element to be a dictionary that has a `workloadVersion` key/value, so that we can add additional data (such as a description of what's updated, or what version of something like XCode is supported).

Reply:
I asked @baronfel what format he'd want for json a couple weeks ago, and he said he just wanted a JSON array of version strings.

I do think there are other things users could potentially want to know, but the key things (from my perspective) are things we'd expect the user to ask for more precisely in a future command. As an example, it's very natural for the user to want to know what workload versions are part of this random version we just gave them, but then they'd run `dotnet workload search version 9.0.103` or whatever it is, and they'd get that information.

I do think having it be parseable as a csv is useful, but since both you and joeloff commented on that, I can make that an option like json and have the default (or typos) put versions on their own lines.

Reply:
Yeah - for _this command_ which is just answering the question "which workload set versions are available", a simple list/json array makes sense.

IMO @dsplaisted's idea is much more aligned with the other commands we have been discussing - "what's in workload set X?" and "what components are supported in X?".

Reply:
I do agree with 

> I don't think CSV with no spaces is great as a default. If the default is supposed to be human-readable, I would suggest outputting it in text format where each version is on a separate line. Eventually this might change to be a table.

though - @Forgind please capture screenshots/examples and put them in the PR description to help discussions here.

Reply:
Thanks - this was very helpful for me to see!